docs/installing_dependencies.adoc

```diff
@@ -46,8 +46,8 @@ To install g2p-seq2seq :
     sudo python setup.py install
 
 
-*I have incorporated above steps into a bash script located inside `install/` directory, that you may simply call using :*
+*I have incorporated above the steps into a bash script located inside `install/` directory. You may call it simply by using :*
```

Comment:
I think that you should swap around "above" and "the steps", that makes more sense :)